src/components/SortingOptionsList.js

```diff
@@ -1,10 +1,19 @@
 import { Checkbox, FormControlLabel, FormGroup, Slider, Typography } from '@mui/material';
-import React, { useContext } from 'react';
+import React, { useContext, useEffect } from 'react';
 import { SortingSettingsContext } from '../contexts/SortingSettingsContext';
 
 export default function SortingOptionsList() {
   const {sortingSpeed, setSortingSpeed, arrayLength, setArrayLength, invertBars, setInvertBars} = useContext(SortingSettingsContext);
 
+  useEffect(() => {
+    setInvertBars(localStorage.getItem('invert') === 'true' ? true : false);
```

Comment:
`true : false` can be omitted.
```suggestion
    setInvertBars(localStorage.getItem('invert') === 'true');
```